src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java

```diff
@@ -75,15 +74,25 @@ public void setRotationAngle(Cuboid cuboid, float x, float y, float z) {
 		cuboid.yaw = y;
 		cuboid.roll = z;
 	}
-	
+
+	@Override
+	public void animateModel(CrowEntity entity_1, float float_1, float float_2, float float_3) {
+		super.animateModel(entity_1, float_1, float_2, float_3);
+	}
```

Comment:
Unnecessary override?

Reply:
Ah, my bad. I thought that was in the class before the PR, guess I added it myself.